### ISSUE TITLE
Getting Started docs

### ISSUE THREAD

**Issue by mangecoeur** [mathjax/MathJax-node#100]
currently getting started means rummaging through the bin/ scripts. would be handy to have a basic useage example in the Readme

**Comment by pkra** [comment]
Thanks for your comment. Do you have a suggestion what would have helped you now that you've rummaged around?

Would shortened versions of the examples in `\bin` have helped? Something like

```
// a simple TeX-input example
var mjAPI = require("./lib/mj-single.js");
mjAPI.config({
  MathJax: {
    // traditional MathJax configuration
  }
});
mjAPI.start();

var yourMath = 'some TeX';

mjAPI.typeset({
  math: yourMath,
  format: "TeX", // "inline-TeX", "MathML"
  mml:true, //  svg:true,
}, function (data) {
  if (!data.errors) {console.log(data.mml)}
});

**Comment by dpvc** [comment]
I think the OP may be saying that we need documentation on how to use the tools in the bin directory (not how to call the API, though we need that, too).  I'm wondering if saying that these all have a `--help` option that gives the usage and available options would due sufficient for that?  And saying that they are mainly there as examples of how to call the API and that they are not all intended as useful in and of themselves.

**Comment by mangecoeur** [comment]
@dpvc  actually i did mean the API (though the things in bin would be handy too) - what I meant was that to learn how to use the API I had to look at the bin/ scripts - it would be handy to have a minimal example in the readme, such as the example @pkra suggested.

**Comment by pkra** [comment]
@dpvc good point; we should add `--help` as well.

@mangecoeur thanks for the clarification.

Here's one for mj-page.

```javascript
var mjAPI = require("./lib/mj-page.js");
var jsdom = require("jsdom").jsdom;

var document = jsdom("<!DOCTYPE html><html lang='en'><head><title>Test</title></head><body>\\[f: X \\to Y\\], where \\( X = 2^{\mathbb{N}}\\)</body></html>");

mjAPI.start();

mjAPI.typeset({
  html: document.body.innerHTML,
  renderer: "NativeMML",
  inputs: ["TeX"],
  xmlns: "mml"
}, function(result) {
  "use strict";
  document.body.innerHTML = result.html;
  var HTML = "<!DOCTYPE html>\n" + document.documentElement.outerHTML.replace(/^(\n|\s)*/, "");
  console.log(HTML);
});
```

**Comment by dpvc** [comment]
OK, @mangecoeur, thanks for the clarification.  @pkra, thanks for the examples.

**Comment by pkra** [comment]
Fixed in develop by #103.
